sequencer/src/context.rs

```diff
@@ -132,8 +155,21 @@ impl<N: network::Type, Ver: StaticVersionType + 'static> SequencerContext<N, Ver
         };
         ctx.spawn(
             "main event handler",
-            handle_events(events, persistence, ctx.state_signer.clone()),
+            handle_events(
+                events,
+                persistence,
+                ctx.state_signer.clone(),
+                event_streamer.clone(),
+            ),
         );
+        // spwan the task for running the event streaming api
+        if let Some(event_streamer) = event_streamer {
+            ctx.spawn(
+                "event streaming api",
+                run_hotshot_event_streaming_api(events_streaming_server.unwrap(), event_streamer),
```

Comment:
From the implementation of `init` it looks like `event_streamer` could be `Some` even when `events_streaming_server` is `None`. Is it possible for this `unwrap` to panic?

Reply:
d1ba612c74d0131e4b06907bd3b5dd3b96aad345